internal/lsp/server.go

```diff
@@ -873,7 +884,6 @@ func (l *LanguageServer) handleWorkspaceDidDeleteFiles(
 		}
 
 		l.diagnosticRequestFile <- evt
-		l.builtinsPositionFile <- evt
```

Comment:
I'm not super familiar with the code base, so please enlighten me: why do we no longer need to inform the builtins-positions thing?

Reply:
There are no builtin positions in a deleted file :)

Reply:
....and the builtins-positions-thingy doesn't have a cache that needs updating?

Reply:
Ah OK. Thanks!